Lib/calendar.py

```diff
@@ -726,6 +724,9 @@ def main(args):
     locale = options.locale, options.encoding
 
     if options.type == "html":
+        if options.month:
+            parser.error("incorrect number of arguments")
```

Comment:
This could include a suggestion to the user on what to change?
```suggestion
            parser.error("Incorrect number of arguments.")
```

Reply:
While this sounds reasonable, it's not the actual point of the error message.
You can imagine it as an `argparse` extension, therefore the error message should be in the same format (your suggested change breaks it) and in line with other `argparse` errors (to answer your question).

Also, I think this is a bit outside the scope of this PR, and in case we would like to do such a change, should it probably be done in a separate PR?

Reply:
You're right on two counts -- I didn't look for enough context on this one.

A